code/modules/modular_armor/armor_pieces.dm

```diff
@@ -171,22 +168,18 @@
 	return ..()
 
 /obj/item/armor_module/armor/arms/marine
-	name = "\improper Jaeger Pattern Medium Infantry arm plates"
-	desc = "Designed for use with the Jaeger Combat Exoskeleton. It provides moderate protection and encumbrance when attached and is fairly easy to attach and remove from armor. Click on the armor frame to attach it. This armor appears to be marked as a Infantry armor piece."
+	name = "\improper Jaeger Pattern Infantry arm plates"
+	desc = "Designed for use with the Jaeger Combat Exoskeleton. It provides protection and encumbrance when attached and is fairly easy to attach and remove from armor. Click on the armor frame to attach it. This armor appears to be marked as a Infantry armor piece."
 	icon_state = "infantry_arms"
-	soft_armor = list("melee" = 9, "bullet" = 12.5, "laser" = 12.5, "energy" = 10, "bomb" = 10, "bio" = 10, "rad" = 10, "fire" = 10, "acid" = 9)
-	slowdown = 0.15
+	soft_armor = list("melee" = 10, "bullet" = 10, "laser" = 10, "energy" = 10, "bomb" = 10, "bio" = 10, "rad" = 10, "fire" = 10, "acid" = 10)
```

Comment:
You are removing all variety from the arm options. Is this desired?

Reply:
yeah, rengusta told to put it like that, same with legs

Reply:
Can confirm, helps with balance and keeping slowdown consistent and reasonable across armors